v_2_0_0/cloudconfig_test.go

```diff
@@ -99,7 +99,6 @@ func TestCloudConfigTemplating(t *testing.T) {
 		if err := cloudConfig.ExecuteTemplate(); err != nil {
 			t.Fatalf("%v: unexpected error templating cloud config: %v", index, err)
 		}
-
```

Comment:
That does not change anything for the guest clusters but in the future please lets make sure we do not touch "frozen" versions at all.